### PR TITLE
binary_gender_bias_mitigated_roberta_snli

### PR DESCRIPTION
**Pull request by ArjunSubramonian**
added endpoint for binary_gender_bias_mitigated_roberta_snli model

depends on: https://github.com/allenai/allennlp-models/pull/268

### CODE REVIEW

File: api/allennlp_demo/binary_gender_bias_mitigated_roberta_snli/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+git+git://github.com/allenai/allennlp.git@v2.5.0
+git+git://github.com/allenai/allennlp-models.git@v2.5.0
```
Comment (epwalsh) [inline api/allennlp_demo/binary_gender_bias_mitigated_roberta_snli/requirements.txt:2]: Might as well just use the PyPI release:

```suggestion
allennlp==2.5.0
allennlp-models==2.5.0
```

Better yet, if we update the default dependencies to point to AllenNLP v2.5.0 (https://github.com/allenai/allennlp-demo/blob/main/api/requirements.txt), then we can just use the default Dockerfile and can remove this requirements file as well as the Dockerfile next to it.

Comment (epwalsh) [reply api/allennlp_demo/binary_gender_bias_mitigated_roberta_snli/requirements.txt:2]: PR: https://github.com/allenai/allennlp-demo/pull/1021. Hopefully this doesn't break anything 😬

**Review by epwalsh — COMMENTED** [review]
I'm having trouble getting this running locally. I'm running:

```bash
./demo start
./demo start binary_gender_mitigated_roberta_snli
```

And all is good until a hit "Run" on the UI, at which point I see this error: 
image

`./demo logs --follow ui` doesn't show an additional information.

@jonborchardt any ideas?

**Comment by ArjunSubramonian** [comment]
> I'm having trouble getting this running locally. I'm running:
> 
> ```shell
> ./demo start
> ./demo start binary_gender_mitigated_roberta_snli
> ```
> 
> And all is good until I hit "Run" on the UI, at which point I see this error:
> image
> 
> `./demo logs --follow ui` doesn't show an additional information.
> 
> @jonborchardt any ideas?

Fixed!

**Review by epwalsh — APPROVED** [review]
This looks great! I just have one suggestion: add an example to the drop-down of default examples that highlights how this model is less biased wrt to gender.

**Comment by ArjunSubramonian** [comment]
> This looks great! I just have one suggestion: add an example to the drop-down of default examples that highlights how this model is less biased wrt to gender.
> 
> image

I merged a separate PR for this in allennlp-models: https://github.com/allenai/allennlp-models/pull/272/files.

**Pull request merged**